CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+## Running
+
+functionless
+
+> yarn install
+
+> yarn watch
+
+test-app
+
+> cd test-app
+
+> yarn install
+
+> yarn watch
+
+website
+
+> cd website
+
+> yarn start
+
+### Testing
+
+Required:
+* Docker - `sudo dockerd`
+* [Localstack](https://docs.localstack.cloud/get-started/#localstack-cli)
+
+`yarn test` - boots up localstack, runs all tests, stops localstack
+`yarn test:fast` - runs all non-localstack tests
+`yarn watch` - watches all tests
+`vs-code jest plugin` - use the vs code jest plugin to run individual tests and debug
+ 
+When using any method that does not invode `yarn test`, `./script/localstack` may be used to start localstack (and boostrap CDK) manually. 
+
+## Additions
+
+### Error Codes
+
+Functionless maintains well documented and linked error codes for compiler and synth errors.
+
+#### Error Code Strategy
+
+https://github.com/functionless/functionless/issues/182#issuecomment-1155683168
+
+TLDR; keep it simple and follow Typescript/React
+
+1.  Error Codes - Start at 10000
```

Comment:
We haven't yet updated our error code numbers to reflect this

Reply:
Doing it as part of this change (see pr description)